resources/www-root/js/league.js

```diff
@@ -174,12 +184,24 @@ var LEAGUE = (function () {
                     if (totalScore === 19 && lost(p.name, g)) { p.brink.P += 1; p.brink.L += 1; }
                     if (g.meta.rhino && won(p.name, g)) { p.badges.push("RHINO!"); }
                     if (g.meta.rhino && lost(p.name, g)) { p.badges.push("RHINOED!"); }
+                    if (isAttacker(p.name, g) && won(p.name, g)) { p.attack.P += 1; p.attack.W +=1; }
+                    if (isAttacker(p.name, g) && lost(p.name, g)) { p.attack.P += 1; p.attack.L +=1; }
+                    if (isDefender(p.name, g) && won(p.name, g)) { p.defend.P += 1; p.defend.W +=1; }
+                    if (isDefender(p.name, g) && lost(p.name, g)) { p.defend.P += 1; p.defend.L +=1; }
                 });
 	    
                 if (p.brink.P > 0) {
                     if (p.brink.W / p.brink.P > 0.6) { p.badges.push("STEADY NERVE"); }
                     if (p.brink.W / p.brink.P < 0.4) { p.badges.push("CRUMBLES"); }
                 }
+                if (p.attack.P > 0) {
+                    if (p.attack.W / p.attack.P > 0.6) { p.badges.push("STRIKER"); }
```

Comment:
Interestingly, this is different to what I had imagined from your description. I was expecting something like `if (p.attack.W / p.W > 0.6)  {....}` so that a player could know if they are personally better in attack or defence.  This badge shows whether they are good in attack/defence compared to everyone else so I suspect the top players will be STRIKER/SHIELD WALL and the worse players will be BANANAFOOT/COLANDER (nice name BTW!)